lib/helpbase.gi

```diff
@@ -128,7 +128,7 @@ BindGlobal( "TRANSATL", MakeImmutable(
 ##
 ##  This approach may suggest wrong spellings for topics containing the 
 ##  substring "Size" or "size", since it's not possible to detect whether 
-##  "size" is a part of another word or a word itselg (e.g. both spellings 
+##  "size" is a part of another word or a word itself (e.g. both spellings
```

Comment:
Same exists in `hpcgap/lib/helpbase.gi`

Reply:
Yes. I have now made the changes also there.